generated/ruby/google-cloud-monitoring/google-cloud-monitoring-v3.gemspec

```diff
@@ -0,0 +1,28 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |s|
+  s.name          = 'google-cloud-monitoring-v3'
```

Comment:
The package name won't contain the major API version, so it's `google-cloud-monitoring` and that's it. We should fix packman template first, but you could manually modify this.